shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/STS/Chain.hs

```diff
@@ -181,6 +181,7 @@ instance
     | BbodyFailure !(PredicateFailure (BBODY crypto))
     | TickFailure (PredicateFailure (TICK crypto))
     | PrtclFailure !(PredicateFailure (PRTCL crypto))
+    | PrtclSeqFailure !(PrtlSeqFailure crypto)
```

Comment:
Are these names alright?

I think it would be slightly cleaner if we made a separate rule for this, unrelated to `PRTCL`. That might make it fit better in the spec? Anyway, I have enough work on the consensus side with the hard fork combinator that needs to be finished soon :slightly_smiling_face:.

Reply:
the names are fine by me. we can make a separate rule later, no need to block this PR.